### ISSUE TITLE
how to undelegate resources auto in EOS but not in hand? 

### ISSUE THREAD

**Issue by csquan** [EOSIO/eos#8396]
how to undelegate resources auto in EOS but not in hand?

**Comment by jgiszczak** [comment]
Can you rephrase the question?

**Comment by csquan** [comment]
if  undelegate resources ,we should do some command in hand,if there any way to do it automatic.
when delegate resources,eos can undelegate resources automatic?

**Comment by jgiszczak** [comment]
When unstaking, the system does currently attempt to schedule a deferred transaction to return the unstaked amount automatically.  This transaction frequently fails, and deferred transactions will eventually be removed entirely.  At that point there will be no internally automated mechanism to recover unstaked tokens once they're unlocked.  There are third party services offering scheduled transactions on EOS-based networks.  I can't say how well they work.
